proposals/low-level-struct-improvements.md

```diff
@@ -112,7 +112,23 @@ ref struct ReadOnlyExample
 }
 ```
 
-A `readonly ref struct` will require that `ref` fields are marked as `readonly ref`. There is no requirement that they are marked as `readonly ref readonly`. This does allow a `readonly struct` to have indirect mutations via such a field but that is no different than a `readonly` field that pointed to a reference type today.
+A `readonly` on a `ref struct` will instance will force `ref` fields to be `readonly ref readonly`. This also means a `readonly ref struct` must mark `ref` fields as `readonly ref readonly`. This maintains the invariant that a `ref struct` which is `readonly` cannot store ref-like state.
```

Comment:
>A `readonly` on a `ref struct` will instance will force `ref` fields to be `readonly ref readonly`. This also means a `readonly ref struct` must mark `ref` fields as `readonly ref readonly`. [](http://example.com/codeflow?start=0&length=190)

The first two sentences in this paragraph seem near-duplicates

Reply:
The first is describing how `readonly` on a value will impact `ref` fields, the other is describing how `readonly` on a declaration will do so. Let me try and clean that up a bit.